server/game/cards/02_SHD/upgrades/Foundling.ts

```diff
@@ -0,0 +1,21 @@
+import { UpgradeCard } from '../../../core/card/UpgradeCard';
+import { EffectName, RelativePlayer } from '../../../core/Constants';
+import { OngoingEffectBuilder } from '../../../core/ongoingEffect/OngoingEffectBuilder';
+
+export default class Foundling extends UpgradeCard {
+    protected override getImplementationId () {
+        return {
+            id: '7687006104',
+            internalName: 'foundling',
+        };
+    }
+
+    public override setupCardAbilities () {
+        this.addConstantAbilityTargetingAttached({
+            title: 'Give the Mandalorian trait to the attached card',
+            ongoingEffect: OngoingEffectBuilder.card.static(EffectName.AddTrait, 'mandalorian'),
```

Comment:
Sorry, missed this before - should use the enum `Trait.Mandalorian` and not the string